Project-Hotpot/Project-Hotpot/Parse-Backend/PFPlaylist.swift

```diff
@@ -120,6 +119,42 @@ class PFPlaylist: PFObject, PFSubclassing {
             }
         })
     }
+    
+    class func getSongPlaylistObject(song: PFSong, playlist: PFPlaylist, completion: @escaping (_ result: Result<PFObject, Error>) -> Void) {
```

Comment:
Nit: Don't need to suffix "Object". E.g. If you have a class called "Car" its more readable to have a method like "getCar()".